app/services/cocina_dro_access.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# This transcodes the access from the item set rights form to Cocina DROAccess schema
+class CocinaDROAccess
+  extend Dry::Monads[:maybe]
+
+  # @param [String] the rights representation from the form (must be one of the keys in Dor::RightsMetadataDS::RIGHTS_TYPE_CODES, or 'default')
```

Comment:
Won't this tell YARD that the name of the param is `the` instead of `rights`?

Reply:
fixed